lib/ProductOpener/I18N.pm

```diff
@@ -120,6 +157,24 @@ sub read_po_files {
 # Separate the singular & plural entries from a hash, as returned by
 # read_po_files(), into two hashes. Obviously returned as two hashrefs.
 #
+
+=head1 FUNCTIONS
```

Comment:
I think we can remove this header, as there is already another =head1 FUNCTIONS header above